docker/linux/startServer.sh

```diff
@@ -218,6 +218,3 @@ export OrigamSettings_DbUsername
 export OrigamSettings_DbPassword
 ./updateEnvironment.sh
 sudo ./updateEnvironmentRoot.sh
-export ASPNETCORE_URLS="http://+:8080"
-dotnet Origam.Server.dll
-
```

Comment:
So the startServer.sh does not start the server anymore. We should either:
- put "dotnet Origam.Server.dll" back
- rename the file
- get rid of it and put everything into entrypoint.sh